features/custom-user-agent.json

```diff
@@ -0,0 +1,19 @@
+{
+    "_meta": {
+        "description": "Custom user agents used by our native apps that override the default user-agent string.",
+        "sampleExcludeRecords": {
+            "domain": "example.com",
+            "reason": "Two factor auth that verifies device pathes using user agent"
+        }
+    },
+    "settings": {
+        "omitApplicationSites": [
+            "cvs.com",
+            "sovietgames.su",
+            "accounts.google.com",
+            "facebook.com"
+        ],
+        "omitVersionSites": []
```

Comment:
What does this do?

Reply:
In Android they have two levels of exceptions for the UserAgent. 1 where they drop DuckDuckGo from the agent (omit application) and another where they drop the browser version (omit version). iOS does not drop the version, only the application name.

Reply:
I see, I personally was going to push for this kinda thing:

```
"exceptions": [
  { domain: "boop.com", "settings": {omitVersion: true}},
  { domain: "else.com" }
]
```

Reply:
This will significantly increase client side integration complexity as (at least on the Apple platforms) the exceptions structure is tightly defined within the config implementation.

Reply:
This is something we're probably going to want for autofill where settings are going to be granular and we might want per site settings.
Creating distinct URL lists won't scale there overtime I don't think.

Reply:
Also the way I was going to implement this in the extension would just be to merge settings with the domain, which makes the implementation less boilerplate and apply across all features.

Reply:
This seems like a config v3 discussion. IMO there's room for debate as tracker allowlist sets up precedence for keeping this sort of thing outside of exceptions.

Reply:
> This will significantly increase client side integration complexity as (at least on the Apple platforms) the exceptions structure is tightly defined within the config implementation.

What do you think about fixing the Apple platforms rather than have them prevent what otherwise would be a better json structure like the one @jonathanKingston suggested?

Reply:
As discussed on the call - we are all aligned that `exceptions > {domain, settings}` seems like a better pattern moving forward, so let's get rid of the old one (`settings > bla: []`) across the whole config in v3 (we use same pattern in trackingCookies3p, trackerAllowlist, gpc).